CMakeLists.txt

```diff
@@ -179,30 +180,57 @@ check_library_exists(rt shm_open "" LMMS_HAVE_LIBRT)
 
 LIST(APPEND CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}")
 
-FIND_PACKAGE(Qt5 5.9.0 COMPONENTS Core Gui Widgets Xml REQUIRED)
-FIND_PACKAGE(Qt5 COMPONENTS LinguistTools QUIET)
+IF(WANT_QT6)
+	MESSAGE("Building with Qt6 is experimental, not guaranteed to succeed")
+	SET(QT_VERSION_MAJOR 6)
+	SET(WANT_VST OFF)
+	SET(STATUS_VST "VSTs not supported with Qt 6")
+	SET(STATUS_QT6 "ENABLED")
+ELSE()
+	SET(QT_VERSION_MAJOR 5)
+	SET(STATUS_QT6 "NOT ENABLED")
+ENDIF()
+
+FIND_PACKAGE(Qt${QT_VERSION_MAJOR} COMPONENTS Core Gui Widgets Xml REQUIRED)
```

Comment:
This conflicts with #7204 . Though i did fix this with the merge editor, i believe this PR will override that change. Any better way than an if block? @messmerd

Reply:
done with https://github.com/LMMS/lmms/pull/7182/commits/70ede889fe4d9180c0a83927d48625767d121ce3